### PR TITLE
COMMERCE-12308

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/COMMERCE-12308

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/configuration/CPDefinitionOptionRelConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/configuration/CPDefinitionOptionRelConfiguration.java:2]: This is failing on SF

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
LGTM.

Thank you!

File: modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/configuration/CPDefinitionOptionRelConfiguration.java (hunk 1 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/configuration/CPDefinitionOptionRelConfiguration.java:2]: I ran SF several times, but nothing changed. I'll try to update my master branch and retry.

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:test:commerce-functional

**Comment by victoroscuro** [comment]
Failures in the relevant aren't related to the changes made in this PR.

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**